### DISCUSSION TITLE
Are there any GPUs available in Catalyst's Hubs? if yes, how many? / ¿Hay GPUs disponibles para los Hubs de Catalyst? ¿Si es así, cuántos?

### DISCUSSION THREAD

**Discussion by SLLDeC** [czi-catalystproject/czi-catalystproject.github.io#40] [Q&A]
Previous answer:

@jnywong answer:
> Yes, GPUs are available! But again these need to be specifically configured for the hub.

**Comment by jnywong** [comment]
I must also add that GPU hardware is expensive and therefore not offered by default on a Catalyst Project Hub. 

So technically it is possible, but I think there is a separate discussion here about whether we can allocate project funds towards this.
